test/wsdls/stockquote.wsdl

```diff
@@ -42,7 +42,7 @@
     </wsdl:portType>
 
     <wsdl:binding name="StockQuoteSoapBinding" type="tns:StockQuotePortType">
-        <soap:binding style="document" transport="http://schemas.xmlsoap.org/soap/http"/>
+        <soap:binding style="rpc" transport="http://schemas.xmlsoap.org/soap/http"/>
```

Comment:
@raymondfeng yes, the end service is an rpc, but the wsdl wrongly had it as document